main/src/test/java/org/mobilitydata/gtfsvalidator/validator/RouteNameValidatorTest.java

```diff
@@ -122,4 +123,20 @@ public void equalRouteShortNameRouteLongNameAndRouteDescShouldGenerateTwoNotices
         .contains(
             new SameNameAndDescriptionForRouteNotice(1, "r1", "duplicate", "route_short_name"));
   }
+
+  @Test
+  public void nullOrEmptyRouteLongNameShouldGenerateNotice() {
+    assertThat(validateRoute(createRoute("short name", null, "desc")))
+        .containsExactly(new RouteNameMissingOrEmptyNotice(1, "route_long_name"));
+    assertThat(validateRoute(createRoute("short name", "", "desc")))
+        .containsExactly(new RouteNameMissingOrEmptyNotice(1, "route_long_name"));
+  }
+
+  @Test
+  public void nullOrEmptyShortNameShouldGenerateNotice() {
+    assertThat(validateRoute(createRoute(null, "long name", "desc")))
```

Comment:
Same as above but for line 109

Reply:
Indeed, collapsed these tests in https://github.com/MobilityData/gtfs-validator/pull/797/commits/acda7f672e3e0bb58167d297b5f6e3ee7e9c8367.